games/static/games/proto.js

```diff
@@ -163,12 +191,15 @@ class Board {
         this.setup = setup
         this.setupObject = setup.getAlienLocations()
         this.alienLocations = Object.keys(this.setupObject).map(Number)
+        this.initialAlienLocations = this.alienLocations
         this.alienHealthLevels = Object.values(this.setupObject)
```

Comment:
setup must become an array of alien objects
its much easier to understand then key values of locations and health level

Reply:
isnt health level say if alien is invader or intruder ?

Reply:
i have decided to give up the invader/intruder all the aliens are invaders and their css class is determined by their health level